xgboost-predictor/src/main/java/biz/k11i/xgboost/learner/ObjFunction.java

```diff
@@ -86,6 +91,26 @@ public float predTransform(float pred) {
         return pred;
     }
 
+    /**
+     * Logistic regression.
+     */
+    static class RegObjFunction extends ObjFunction {
+        @Override
+        public float[] predTransform(float[] preds) {
```

Comment:
```suggestion
        public float[] predTransform(final float[] preds) {
```

Nitpicking level 10000000.

Reply:
While I agree with your suggestion, I will not apply it. Its good practice to make args final, but doing so IMHO actually makes code worse to read, the only solution is to switch to Kotlin ;-)

Reply:
Of course, I'm more than fine with it.

Reply:
(not talking about the Kotlin switch :rofl:  )